<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginBox</class>
 <widget class="QDialog" name="loginBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login to ShipHaz</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>331</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QLineEdit" name="userBox"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="pwBox"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="guestButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Login as Guest</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>160</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>loginButton</sender>
   <signal>clicked()</signal>
   <receiver>loginBox</receiver>
   <slot>login()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>48</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>guestButton</sender>
   <signal>clicked()</signal>
   <receiver>loginBox</receiver>
   <slot>guest()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>loginBox</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>login()</slot>
  <slot>guest()</slot>
 </slots>
</ui>
